src/commands/remove/index.ts

```diff
@@ -25,35 +25,66 @@ registerCommand<PreHandlers>({
 			this.EarlyEnd.InvalidInput('No emote name provided');
 		}
 
-		const emote = (await gql.CurrentEnabledEmotes(EmoteSet())).find(
-			(emote) => emote.name === ctx.input[0],
-		);
+		ctx.input = ctx.input.reduce((args: string[], arg: string) => {
```

Comment:
Unnecessary to add the argument types here.

Reply:
![image_78](https://user-images.githubusercontent.com/18603393/225698885-19bc8caa-1cd3-4c32-ac1a-246d7a5a8607.png)
Uhh you sure about that?

Reply:
Toy language